_rules/lang-valid-de46e4.md

```diff
@@ -23,7 +23,7 @@ This rules applies to each HTML or SVG element that is a [descendant](https://do
 
 ## Expectation
 
-The `lang` and `xml:lang` attributes of the test target have a [valid language subtag](#valid-language-subtag) if the attribute is not empty ("").
+For each test target, the `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) unless the attribute is empty ("").
```

Comment:
I disagree. The statement is still true if the target only has a `lang` attribute, but I agree this can probably be worded better. Given that this is the original wording of the rule I'm using, I think this is out of scope for this change. Feel free to open a separate issue for this.